### PR TITLE
Sphinx RST File Modification - Added QueryProfiler

### PR DESCRIPTION
**Pull request by mail4umar**
Also minor:
- Added "okwarning" in docstring for regexp functions

### CODE REVIEW

File: verticapy/sql/geo/index.py [hunk 1]
```diff
@@ -256,15 +256,15 @@ def rename_index(source: str, dest: str, overwrite: bool = False) -> bool:
...
+        from verticapy.sql.geo import rename_index, describe_index
 
         # Describes all indexes
         describe_index()
 
     .. ipython:: python
         :suppress:
 
-        from verticapy.geo import *
+        from verticapy.sql.geo import *
```
Comment (oualib) [inline verticapy/sql/geo/index.py:267]: Please do not use *, import what's needed.

File: verticapy/sql/geo/index.py [hunk 2]
```diff
@@ -163,15 +163,15 @@ def describe_index(
...
+        from verticapy.sql.geo import describe_index
 
         # Describes all indexes
         describe_index()
 
     .. ipython:: python
         :suppress:
 
-        from verticapy.geo import *
+        from verticapy.sql.geo import *
```
Comment (oualib) [inline verticapy/sql/geo/index.py:174]: Please do not use *, import what's needed.

File: verticapy/sql/geo/index.py [hunk 3]
```diff
@@ -90,7 +90,7 @@ def create_index(
...
```
Comment (oualib) [inline verticapy/sql/geo/index.py:93]: Please do not use *, import what's needed.

File: verticapy/sql/geo/functions.py [hunk 4]
```diff
@@ -331,7 +331,7 @@ def split_polygon_n(p: str, nbins: int = 100) -> vDataFrame:
     .. ipython:: python
         :suppress:
 
-        from verticapy.geo import *
+        from verticapy.sql.geo import *
```
Comment (oualib) [inline verticapy/sql/geo/functions.py:334]: Please do not use *, import what's needed.

File: verticapy/sql/geo/functions.py [hunk 5]
```diff
@@ -199,7 +199,7 @@ def intersect(
...
```
Comment (oualib) [inline verticapy/sql/geo/functions.py:202]: Please do not use *, import what's needed.

File: verticapy/sql/geo/functions.py [hunk 6]
```diff
@@ -77,7 +77,7 @@ def coordinate_converter(
...
```
Comment (oualib) [inline verticapy/sql/geo/functions.py:80]: Please do not use *, import what's needed.

File: verticapy/sql/functions/regexp.py [hunk 7]
```diff
@@ -350,16 +355,17 @@ def regexp_substr(
...
 
     .. ipython:: python
         :suppress:
+        :okwarning:
 
         from verticapy.datasets import load_titanic
         import verticapy.sql.functions as vpf
         titanic = load_titanic()
-        titanic["title"] = vpf.regexp_substr(titanic["name"], '([A-Za-z])+\. ')
+        titanic["title"] = vpf.regexp_substr(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:368]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 8]
```diff
@@ -350,16 +355,17 @@ def regexp_substr(
         titanic = load_titanic()
         # Apply the regexp_substr function, creating a new "title" column
         titanic["title"] = vpf.regexp_substr(titanic["name"],
-                                            '([A-Za-z])+\. ')
+                                            '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:358]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 9]
```diff
@@ -287,17 +291,18 @@ def regexp_replace(
...
 
     .. ipython:: python
         :suppress:
+        :okwarning:
 
         from verticapy.datasets import load_titanic
         import verticapy.sql.functions as vpf
         titanic = load_titanic()
-        titanic["new_title"] = vpf.regexp_replace(titanic["name"], '([A-Za-z])+\. ', '[title here] ')
+        titanic["new_title"] = vpf.regexp_replace(titanic["name"], '([A-Za-z])+\\.', '[title here] ')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:305]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 10]
```diff
@@ -287,17 +291,18 @@ def regexp_replace(
         titanic = load_titanic()
         # Apply the regexp_replace function, creating a new "new_title" column
         titanic["new_title"] = vpf.regexp_replace(titanic["name"],
-                                                 '([A-Za-z])+\. ',
+                                                 '([A-Za-z])+\\.',
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:294]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 11]
```diff
@@ -222,16 +225,17 @@ def regexp_like(expr: SQLExpression, pattern: SQLExpression) -> StringSQL:
...
 
     .. ipython:: python
         :suppress:
+        :okwarning:
 
         from verticapy.datasets import load_titanic
         import verticapy.sql.functions as vpf
         titanic = load_titanic()
-        titanic["has_title"] = vpf.regexp_like(titanic["name"], '([A-Za-z])+\. ')
+        titanic["has_title"] = vpf.regexp_like(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:238]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 12]
```diff
@@ -222,16 +225,17 @@ def regexp_like(expr: SQLExpression, pattern: SQLExpression) -> StringSQL:
 
         titanic = load_titanic()
         # Apply the regexp_like function, creating a new "has_title" column
-        titanic["has_title"] = vpf.regexp_like(titanic["name"], '([A-Za-z])+\. ')
+        titanic["has_title"] = vpf.regexp_like(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:228]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 13]
```diff
@@ -170,17 +172,18 @@ def regexp_instr(
...
 
     .. ipython:: python
         :suppress:
+        :okwarning:
 
         from verticapy.datasets import load_titanic
         import verticapy.sql.functions as vpf
         titanic = load_titanic()
-        titanic["title_start"] = vpf.regexp_instr(titanic["name"], '([A-Za-z])+\. ', return_position = 0)
+        titanic["title_start"] = vpf.regexp_instr(titanic["name"], '([A-Za-z])+\\.', return_position = 0)
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:186]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 14]
```diff
@@ -170,17 +172,18 @@ def regexp_instr(
         titanic = load_titanic()
         # Apply the regexp_instr function, creating a new "title_start" column
         titanic["title_start"] = vpf.regexp_instr(titanic["name"],
-                                                 '([A-Za-z])+\. ',
+                                                 '([A-Za-z])+\\.',
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:175]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 15]
```diff
@@ -104,16 +105,17 @@ def regexp_ilike(expr: SQLExpression, pattern: SQLExpression) -> StringSQL:
...
 
     .. ipython:: python
         :suppress:
+        :okwarning:
 
         from verticapy.datasets import load_titanic
         import verticapy.sql.functions as vpf
         titanic = load_titanic()
-        titanic["has_title"] = vpf.regexp_ilike(titanic["name"], '([A-Za-z])+\. ')
+        titanic["has_title"] = vpf.regexp_ilike(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:118]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 16]
```diff
@@ -104,16 +105,17 @@ def regexp_ilike(expr: SQLExpression, pattern: SQLExpression) -> StringSQL:
 
         titanic = load_titanic()
         # Apply the regexp_ilike function, creating a new "has_title" column
-        titanic["has_title"] = vpf.regexp_ilike(titanic["name"], '([A-Za-z])+\. ')
+        titanic["has_title"] = vpf.regexp_ilike(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:108]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 17]
```diff
@@ -55,16 +55,17 @@ def regexp_count(
...
 
     .. ipython:: python
         :suppress:
+        :okwarning:
 
         from verticapy.datasets import load_titanic
         import verticapy.sql.functions as vpf
         titanic = load_titanic()
-        titanic["has_title"] = vpf.regexp_count(titanic["name"], '([A-Za-z])+\. ')
+        titanic["has_title"] = vpf.regexp_count(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:68]: are you sure?

File: verticapy/sql/functions/regexp.py [hunk 18]
```diff
@@ -55,16 +55,17 @@ def regexp_count(
 
         titanic = load_titanic()
         # Apply the regexp_count function, creating a new "has_title" column
-        titanic["has_title"] = vpf.regexp_count(titanic["name"], '([A-Za-z])+\. ')
+        titanic["has_title"] = vpf.regexp_count(titanic["name"], '([A-Za-z])+\\.')
```
Comment (oualib) [inline verticapy/sql/functions/regexp.py:58]: are you sure?

**Review by oualib — CHANGES_REQUESTED** [review]
Two main comments:
 - avoid "import *", import only the needed functions
 - Double check this double backslash, not sure it is correct.

File: verticapy/sql/functions/regexp.py (hunk 14 above)
Comment (mail4umar) [reply verticapy/sql/functions/regexp.py:175]: It seems that VerticaPy does not like the single backslash:
Capture

**Review by oualib — APPROVED** [review]

**Pull request merged**